src/overlaybd/fs/zfile/compressor.cpp

```diff
@@ -76,17 +76,165 @@ class Compressor_lz4 : public ICompressor {
     }
 };
 
+#ifdef ENABLE_QAT
```

Comment:
add a member 'qat_enable' in Compressor_lz4 and move 'bool check_qat()' in this file?

Reply:
ok, good point